sql/parse/parse.go

```diff
@@ -413,8 +413,10 @@ func convertCreateView(ctx *sql.Context, c *sqlparser.DDL) (sql.Node, error) {
 
 	queryAlias := plan.NewSubqueryAlias(c.View.Name.String(), queryNode)
 
+	selectStr := query[c.ViewSelectPositionStart:c.ViewSelectPositionEnd]
```

Comment:
Without a test case it's hard to know if it's an off-by-one error for the end index.